### DISCUSSION TITLE
Help for achievements

### DISCUSSION THREAD

**Discussion by M4rcxs** [Schweinepriester/github-profile-achievements#333] [Q&A] [answered]
How do i get more achievements?

**Comment by bacca87** [comment, marked as answer]
just accept my answer

**Comment by WHAHA-HA** [comment]
I think you can achieve Quickdraw and Pull Shark badges.
If you have something unclear about these achievements, feel free to ask me anytime.
PS: It'd be appreciated if you could mark my answer
